doc/user/filesystem.md

```diff
@@ -207,10 +264,10 @@ To create a new container and link it into the namespace of an existing one,
 use the following command.
 
 ```bash
-$ daos container create --type POSIX --pool <pool uuid> --path <path to entry point>
+$ daos container create --type POSIX <pool_label> --path <path to entry point>
```

Comment:
probably better to move the label after the create, before --type posix

Reply:
fixed.